langchain4j/src/test/java/dev/langchain4j/service/common/AbstractAiServiceWithToolsIT.java

```diff
@@ -786,4 +809,213 @@ protected void should_execute_tool_with_uuid_parameter(ChatModel model) {
             assertThat(toolSpecifications.get(0)).isEqualTo(ToolWithUUIDParameter.EXPECTED_SPECIFICATION);
         }
     }
+
+    @ParameterizedTest
+    @MethodSource("models")
+    protected void should_execute_normal_tool_with_primitive_parameters() {
```

Comment:
``
```suggestion
    protected void should_execute_normal_tool_with_primitive_parameters(ChatModel chatModel) {
```
Because of `@MethodSource`.

Same for other tests